commit cf7e273f6dac76544e3fa3b4ac9c1e9ed96d8c92 (refs/changes/52/1152/1)
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2016-12-20T16:58:20-08:00 (2 years, 10 months ago)
    
    MANTA-3060 muskie cueballs not using bootstrap mode due to typo

diff --git a/etc/config.coal.json b/etc/config.coal.json
index cd5dd04..1649cee 100644
--- a/etc/config.coal.json
+++ b/etc/config.coal.json
@@ -32,7 +32,7 @@
         }
     },
     "cueballHttpAgent": {
-        "resolver": ["nameservice.coal.joyent.us"],
+        "resolvers": ["nameservice.coal.joyent.us"],
         "initialDomains": [
           "authservice.coal.joyent.us"
         ],
@@ -82,7 +82,7 @@
         },
         "spares": 2,
         "maximum": 2000,
-        "resolver": ["nameservice.coal.joyent.us"]
+        "resolvers": ["nameservice.coal.joyent.us"]
     },
     "authToken": {
         "salt": "C93A670ACC05C166",
diff --git a/main.js b/main.js
index 9971302..b22766d 100644
--- a/main.js
+++ b/main.js
@@ -211,7 +211,7 @@ function createCueballHttpAgent(cfg, sharkCfg) {
 
     /* Used only for connections to sharks. */
     var sharkCueball = {
-        resolver: sharkCfg.resolver,
+        resolvers: sharkCfg.resolvers,
         spares: sharkCfg.spares,
         maximum: sharkCfg.maximum,
         linger: sharkCfg.maxIdleTime,
diff --git a/sapi_manifests/muskie/template b/sapi_manifests/muskie/template
index ea6f1ba..02b92a6 100644
--- a/sapi_manifests/muskie/template
+++ b/sapi_manifests/muskie/template
@@ -39,7 +39,7 @@
   "cueballHttpAgent": {
 
     {{! Use "bootstrap" mode to find binder, then use it for lookups. }}
-    "resolver": ["nameservice.{{DOMAIN_NAME}}"],
+    "resolvers": ["nameservice.{{DOMAIN_NAME}}"],
 
     {{! Pre-populate pools to make startup faster. }}
     "initialDomains": [
@@ -105,7 +105,7 @@
     },
 
     {{! Use bootstrap mode to find binder, same as for cueballHttpAgent. }}
-    "resolver": ["nameservice.{{DOMAIN_NAME}}"],
+    "resolvers": ["nameservice.{{DOMAIN_NAME}}"],
 
     {{! TCP keepalive initial timeout. }}
     "maxIdleTime": 10000,
